From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Steven Barrett <steven@liquorix.net>
Date: Wed, 11 Aug 2021 18:47:46 -0500
Subject: [PATCH] ZEN: INTERACTIVE: Add help text for the MG-LRU tweaks

The tweaks themselves get merged with the MG-LRU patchset.
---
 init/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/init/Kconfig b/init/Kconfig
index 3a89cebdd80f756a0d5d7a553d64b463e0ec1365..bf583d6054804b213f3d6a484403ee3197b8760b 100644
--- a/init/Kconfig
+++ b/init/Kconfig
@@ -124,6 +124,7 @@ config ZEN_INTERACTIVE
 	  --- Virtual Memory Subsystem ---------------------------
 
 	    Background-reclaim hugepages...:   no   ->   yes
+	    MG-LRU minimum cache TTL.......:   0    ->   1000 ms
 
 config BROKEN
 	bool
